astropy/stats/bls/__init__.py

```diff
@@ -1,14 +1,41 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
-"""
-Box Least Squares
-=================
+# The BoxLeastSquares periodogram functionality has been moved to
+# astropy.timeseries.bls. The purpose of this file is to provide backward-
+# compatibility during a transition phase. We can't emit a deprecation warning
+# simply on import of this module, since the classes are imported into the
+# top-level astropy.stats, so instead we wrap the main class and emit a
+# warning during initialization.
 
-AstroPy-compatible reference implementation of the transit periorogram used
-to discover transiting exoplanets.
+import warnings
 
-"""
+from astropy.timeseries.bls import (BoxLeastSquares as OriginalBoxLeastSquares,
+                                    BoxLeastSquaresResults as OriginalBoxLeastSquaresResults)
+from astropy.utils.exceptions import AstropyDeprecationWarning
 
-__all__ = ["BoxLeastSquares", "BoxLeastSquaresResults"]
+__all__ = ['BoxLeastSquares', 'BoxLeastSquaresResults']
 
-from .core import BoxLeastSquares, BoxLeastSquaresResults
+
+class BoxLeastSquares(OriginalBoxLeastSquares):
```

Comment:
what about `OriginalBoxLeastSquares` --> `TimeseriesBoxLeastSquares`? after all this module was the orginal.

Reply:
Fixed